### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mckinnsb**
The wording may not be the best, but I think there should be a mention to codeclimate init --upgrade, as it was not mentioned and I had to actually contact someone at CodeClimate to discover how to make the output from the web application match the CLI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,16 @@ brew upgrade codeclimate
...
 ```
+### Updating existing Code Climate configuration
+
+If you are already using Code Climate and have a .codeclimate.yml file within your repository, you will probably want to run:
+
+```
+codeclimate init --upgrade
+```
+
+This will generate .yml files that will enable CodeClimate CLI to run with the configuration that is automatically generated by the Code Climate webapp, ensuring that the output matches between the two.
```
Comment (GordonDiggs) [inline README.md]: 💄 The product name should be written as "Code Climate"

**Comment by mckinnsb** [comment]
Sorry, I am a little confused by "can you also address the issues that Code Climate found? Thank you!"

**Comment by GordonDiggs** [comment]
@mckinnsb The PR status is red from Code Climate. Here's a link to the results https://codeclimate.com/github/codeclimate/codeclimate/pull/571

**Comment by mckinnsb** [comment]
It's passing Code Climate now - how's this look?

**Comment by mckinnsb** [comment]
@GordonDiggs Hey, how's this look?

**Comment by mckinnsb** [comment]
woo!

**Pull request merged**